entregables/site/Docs/Manual-de-Usuario/Evaluaciones.markdown

```diff
@@ -0,0 +1 @@
+# Evaluaciones
```

Comment:
Agree,  and I would remove related TODO mark in controller.
El 10/06/2015 23:03, "Alpha" <notifications@github.com> escribió:

> In entregables/site/Docs/Manual-de-Usuario/Evaluaciones.markdown
> <https://github.com/CommonJobs/CommonJobs/pull/375#discussion_r32184690>:
>
> > @@ -0,0 +1 @@
> > +# Evaluaciones
>
> I would add a //TODO here, or something that is searchable so that it is
> easy later to find out everything that's missing to be completed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CommonJobs/CommonJobs/pull/375/files#r32184690>.
>